rasa/core/training/interactive.py

```diff
@@ -966,10 +969,10 @@ async def _predict_till_next_listen(
     listen = False
     while not listen:
         result = await request_prediction(endpoint, conversation_id)
-        predictions = result.get("scores")
+        predictions = result.get("scores") or []
```

Comment:
how about raising an exception in case `predictions` is empty?

Reply:
`RasaCoreException` or something more specific?

Reply:
Added `RasaCoreException`. Can we add any more specific information in the message, for example: Why such a situation may arise?

Reply:
Do we have something like an invalid config exception? I'd say that they need to retrain their model with a policy specified in the config then?

Reply:
Added.